app/models/miq_server/environment_management.rb

```diff
@@ -29,7 +29,7 @@ def minimal_env_options
         options.shift # remove the "minimal" from the front of the array
 
         # Special case where Netbeans is handling the UI worker for debugging
-        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting noui) : o }.flatten
+        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting no_ui) : o }.flatten
```

Comment:
As far as this specific command goes I would propose that we make the option something like "debugger" or "ui_debugger" instead of the word "netbeans"